scripts/gui/thrive_style.css

```diff
@@ -1109,6 +1100,16 @@ video {
 }
 
 
+#NucleusIcon {  
+  position: relative; 
+  left: calc(50% - 30px); 
+  width: 60px;  
+  height: 60px; 
+  background-size: contain; 
+  background-image: url("../../Textures/gui/bevel/NucleusIcon.png");  
+}
+
+
```

Comment:
what was th epoint of moving the class here?

Reply:
Actually looks like the plastid being moved didnt register in the diff so you were moving it back to a previous position, so im okay with this.